runtime/smithy-client/common/test/aws/smithy/kotlin/runtime/client/LogModeTest.kt

```diff
@@ -33,7 +33,7 @@ class LogModeTest {
     fun testToString() {
         val mode = LogMode.allModes().reduce { acc, curr -> acc + curr }
         assertTrue { LogMode.allModes().all { mode.isEnabled(it) } }
-        val expected = "LogRequest|LogRequestWithBody|LogResponse|LogResponseWithBody"
+        val expected = "LogRequest|LogRequestWithBody|LogResponse|LogResponseWithBody|LogSigning"
```

Comment:
Could you also add this value to the composite string test on L46? (tests the reverse of this, `String`->`LogMode`)